### ISSUE TITLE
Accessibility:File Uploader: Placeholder text “Select the file” and tooltip information “Choose file” is not conveyed when ‘Browse’ button is accessed.  

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3497]
Description: Placeholder text “Select the file” and tooltip information “Choose file” is not conveyed when ‘Browse’ button is accessed.  

Expected: placeholder text and tooltip information should be conveyed to the user. 

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94651593-50852900-0316-11eb-972e-1cf808f893a4.png)

**Comment by puru-hk** [comment]
As this issue raised by accessibility team. Accessibility team has to verify and close this issues. Hence reopening this issue.

**Comment by Sanjay-1312** [comment]
Hello Colleague,
I can still able to reproduce the same issue.
The in place label is not read.
So please fix the issue
JAWS 2019
@manu-kr can you please fix the issue

Thanks and Regards
Sanjay

**Comment by Sanjay-1312** [comment]
Hello Colleague,
The issue is fixed Hence closing the ticket.
JAWS 2019

Thanks and Regards
Sanjay
